lib/Configuration.py

```diff
@@ -677,9 +680,19 @@ def TryGetNetworkFile(self, file=None, url=None, handler=None,
                         # Do I need to do something different for the progress handler?
                         retval.seek(0)
                         return retval
-                    log.error("Got http error %s" % str(error))
-                    url_exc = error
+                    elif error.response.status_code == HTTP_NOT_FOUND.value:
+                        # The requested file is not found on this server.
+                        url_exc = Exceptions.UpdateNetworkFileNotFoundException("Requested file %s not found" % (file if file else url))
+                        log.error("Error 404: %s" % str(url_exc))
+                    else:
+                        log.error("Got http error %s" % str(error))
+                        url_exc = Exceptions.UpdateNetworkServerException("Unable to load from url %s: " % (url, error.response.status_code))
```

Comment:
Just noticesd we have two arguments for the formatted string but only one %s?

Reply:
Oops.  Fixed.  Also removed a debugging line that had the same issue.